yoda-toolfile.spec

```diff
@@ -15,7 +15,9 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/yoda.xml
     <environment name="LIBDIR" default="$YODA_BASE/lib"/>
     <environment name="INCLUDE" default="$YODA_BASE/include"/>
   </client>
-  <use name="boost"/>
+  <use name="python"/>
+  <use name="cython"/>
+  <use name="root_cxxdefaults"/>
```

Comment:
Hmm, these are the library of root being passed at compilation time:
 -lCore -lRIO -lNet -lHist -lGraf -lGraf3d -lGpad -lTree -lRint -lPostscript -lMatrix -lPhysics -lMathCore -lThread -lMultiProc -pthread -lm -ldl -rdynamic

This said it create a non dynamic executable 'yoda2root' which does not seems to have any dependencies at the end:

-bash-4.1$ ldd /afs/cern.ch/work/x/xjanssen/cms/Rivet/81X_gcc530/CMSSW_8_1_0_pre12/external/slc6_amd64_gcc530/bin/yoda2root
	not a dynamic executable